### DISCUSSION TITLE
Easy single project task running

### DISCUSSION THREAD

**Discussion by AlexAegis** [vercel/turborepo#9339] [Ideas]
### Goals

1. I want to just copy the name of a failed task and run it with `npx turbo`
2. This would mean `npx turbo @org/foo#task` would be equivalent of `npx turbo --filter @org/foo task`

### Background

When running tasks across the repo, some of them might fail and then turbo reports the failed tasks in a list. It would be very convenient for turbo to accept this same reported `package#task` format as input so I can copy it and use it in another command while I fix the failed task.

### Proposal

I'd like turbo to parse the same `package#task` format it outputs, as inputs

**Comment by lishaduck** [comment]
It'd also be useful b/c I need to run multiple specific package's tasks, i.e., `turbo watch package1#build package2#build web#start`.

**Comment by anthonyshew** [comment]
As fate would have it, we just added this last week in [this PR](https://github.com/vercel/turborepo/pull/9344)! It's available in canary today, and will be a part of next release!

**Reply by AlexAegis** [reply]
great minds think alike!
